### DISCUSSION TITLE
Is it possible to read a file from an absolute path?

### DISCUSSION THREAD

**Discussion by cjcrandall** [cypress-io/cypress#9637] [Questions and Help] [answered]
From the docs, `cy.fixture` assumes a relative path from the fixtures folder, and `cy.readFile` assumes a relative path from the project root. But is it possible with either of these, or some other means, to read a file from an absolute file path?

**Comment by bahmutov** [comment]
Only by using cy.task

**Reply by cjcrandall** [reply]
That's an interesting design decision. Do you know what the rationale is or if that might be changed in the future? I can use `../` to navigate _out_ of those default directories but that feels a bit hacky. In my use case, I have `json` files that are part of the production code (so don't belong in a fixtures folder) that I want to load, and I also have multiple cypress projects in my repo.

**Comment by bahmutov** [comment, marked as answer]
Well you could also just import the json file directly since the bundler will bundle spec files and anything they import.

**Reply by cjcrandall** [reply]
Neat, that hadn't occurred to me. I'll try it!

**Comment by jennifer-shehane** [comment]
You can change the path of the [`fixturesFolder`](https://on.cypress.io/configuration#Folders-Files) also, but this would change the destination to a single directory. But there are other ways to import files for use if you need as @bahmutov mentioned.
